docs/nodes/credentials/Mautic/README.md

```diff
@@ -15,20 +15,46 @@ Create an account on a [Mautic](https://www.mautic.org/) instance.
 
 ## Using OAuth
 
-1. Log in to your Mautic account.
-2. Click on the gear icon in the top right.
-3. Select 'API Credentials' in the dropdown menu.
-4. Click on the 'New' button.
-5. Select the 'OAuth 2' option from the *Authorization Protocol* dropdown list.
-6. Copy the 'OAuth Callback URL' provided in the Mautic OAuth2 API credentials in n8n and paste it in the 'Redirect URI' field in the Mautic credential page.
-7. Fill out any other necessary information and click 'Apply'.
-8. Click the *Save & Close* button to save your credentials in Mautic.
-9. Use the displayed 'Client ID' and the 'Client Secret' with your Mautic OAuth API credentials in n8n.
-10. Click on the circle button in the OAuth section to connect a Mautic account to n8n.
-11. Click the *Save* button to save your credentials in n8n.
-
-![Getting Mautic OAuth credentials](./using-oauth.gif)
+1. Open your Mautic dashboard.
+2. Click on the gear icon on the top right and select ***API Credentials***.
+3. Click on the ***+ New*** button on the top right.
+4. Select 'OAuth 2' from the ***Authorization Protocol*** dropdown list.
+5. Enter the name of the credentials in the ***Name*** field.
+6. Copy the 'OAuth Callback URL' provided in the 'Mautic OAuth2 API' credentials in n8n.
+7. Paste it in the ***Redirect URI*** field on the Mautic credentials page.
+8. Click on the ***Apply*** button.
+9. Enter the name for your credentials in the ***Credentials Name*** field in the 'Mautic OAuth2 API' credentials in n8n.
+10. Enter the authorization URL in the ***Authorization URL*** field. Refer to the [FAQs] to learn more about the authorization URL.
```

Comment:
@harshil1712 There are no FAQs in this document + the link isn't there. Can you please fix?

Reply:
Removed the step as it is no longer needed in the newer version.